src/main/java/Duke.java

```diff
@@ -1,10 +1,162 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.time.LocalDate;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.io.FileWriter;
+
```

Comment:
Perhaps you can reorder the import statements such that it complies with the coding standard